keyboards/clawsome/sidekick/rules.mk

```diff
@@ -0,0 +1,33 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```

Comment:
```suggestion
```

Reply:
Since it is advertised as supporting a ProMicro/Elite C, I would rather the list be left here.